include/arch/arm/aarch64/asm_inline_gcc.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Either public functions or macros or invoked by public functions */
+
+#ifndef ZEPHYR_INCLUDE_ARCH_ARM_AARCH64_ASM_INLINE_GCC_H_
+#define ZEPHYR_INCLUDE_ARCH_ARM_AARCH64_ASM_INLINE_GCC_H_
+
+/*
+ * The file must not be included directly
+ * Include arch/cpu.h instead
+ */
+
+#ifndef _ASMLANGUAGE
+
+#include <zephyr/types.h>
+#include <sys/util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DAIFSET_IRQ	BIT(1)
+#define DAIF_MASK	BIT(7)
```

Comment:
how about `DAIF_IRQ_MASK`  as the functions here are specific to IRQ

Reply:
> how about `DAIF_IRQ_MASK` as the functions here are specific to IRQ

It would be even better if we drop `_MASK` in order to align with other definitions.

Reply:
ACK. I'll change that to `DAIF_IRQ` in the next respin. Thanks.